interfaces/HO-Portal/src/app/program/program-people/program-people.component.ts

```diff
@@ -200,38 +328,63 @@ export class ProgramPeopleComponent implements OnChanges {
           pa: `PA #${index + 1}`,
           score: person.score,
           did: person.did,
-          created: formatDate(person.created, this.dateFormat, this.locale),
-          updated: formatDate(person.updated, this.dateFormat, this.locale),
+          processStarted: formatDate(person.created, this.dateFormat, this.locale),
+          digitalIdCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          digitalIdValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          inclusionCommunication: person.inclusionDate ? formatDate(person.inclusionDate, this.dateFormat, this.locale) : null,
+          included: person.included ? 'Included' : (person.excluded ? 'Excluded' : ''),
         };
 
+        this.paymentColumns.map((_, index2) => {
```

Comment:
The lint-rule to not have `shadow variables/parameters` isn't there to force you to pick a "name-I-want+1" as a variable name. ;)

It is to avoid doing to much of the same (looking) things in 1 method.
Everything that happens in a `Array.map`-callback, could be done in its own separate, named method.

That way you can use `index` as you variable/parameter for the array-index.

Reply:
Also, if a `map`-callback doesn't return a value, it isn't really a mapping, but more a `for`-loop/`forEach`-callback.